install/kots/manifests/kots-config.yaml

```diff
@@ -4,9 +4,19 @@ metadata:
   name: gitpod
 spec:
   groups:
+    - name: prerequisites
```

Comment:
Had to separate this out, as it was confusing to be in general settings. Currently, Trying to make this config screen pass, only if the items here are ticked off 🤔